desktop/src/main/java/bisq/desktop/main/offer/takeoffer/TakeOfferDataModel.java

```diff
@@ -173,6 +185,7 @@ protected void activate() {
             tradeManager.checkOfferAvailability(offer,
                     false,
                     () -> {
+                log.warn("OfferAvailability completed!");
```

Comment:
I think something went wrong with commit squashing as the `log.warn` is in again.

Reply:
Thanks!  My bad for debugging using log.warn()